src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -76,6 +76,7 @@ public class KNNSettings {
     public static final String MODEL_INDEX_NUMBER_OF_REPLICAS = "knn.model.index.number_of_replicas";
     public static final String MODEL_CACHE_SIZE_LIMIT = "knn.model.cache.size.limit";
     public static final String ADVANCED_FILTERED_EXACT_SEARCH_THRESHOLD = "index.knn.advanced.filtered_exact_search_threshold";
+    public static final String KNN_FAISS_AVX2_DISABLED = "knn.faiss.avx2.disabled";
```

Comment:
Do we also want to just create two libs for arm64 so users can use the non-simd one as well?

Changing knn.faise.avx2.disabled to knn.faise.simd.disabled is better suited for both architecture?

Reply:
As discussed offline, we want to go with exisiting changes